guides/concepts/index.md

```diff
@@ -1 +1,100 @@
-From an outside perspective it is clear that a lot of the underlying concepts in Cardstack reflect some of the useful architectural decisions of Ember. If you are familiar with Ember then you are likely going to understand the most/all of these concepts without too much effort.
+This section is a crash course in Cardstack concepts. It includes vocabulary that you'll find throughout the other topics, and it draws some parallels to other frameworks and JavaScript in general.
+
+
+## Essential Vocabulary
+
+### Cards
+
+Cards are reusable units of functionality - like a blog article, a text editor, a payment system, and more.
+A developer would use many cards, and cards inside of cards, to create the end user experience.
+Each card can have its own data schema, which flows from the front end to the data layer. When someone is looking at a card, it is reflected in the browser URL.
+The information contained in that URL informs communication with the back end servers.
+
+### The Cardstack mono-repo
+
+The [Cardstack mono-repo](https://github.com/cardstack/cardstack) is a collection of the various elements that could be used to create a Cardstack environment.
+It contains some commonly-used Cards, the Cardstack Hub that connects data sources, and the code that ties everything together behind the scenes.
+
+### Plugins and Data Adapters
+
+A plugin, or data adapter, is made up of four functions needed to connect with a new data source.
+For example, someone could write or use plugins to connect to GitHub, Firebase, and a blockchain source.
+That same plugin code could be used by anyone running a Cardstack environment who wants to connect to the same data source.
+
+### The Hub
+
+The Cardstack Hub is where data from many sources come together.
+Developers can activate any number of plugins provided by Cardstack or insert their own.
+The Cardstack Hub does all the heavy lifting that takes the data retrieved by plugins and makes it available to the front end.
+To connect to a new data source, a developer only has to write a plugin, not modify the Hub. The Hub's code is contained within the Cardstack monorepo.
+
+### The Cardstack Environment
+
+It’s helpful to think of Cardstack “environments” rather than "apps." 
+Although Cardstack is built for the web and not native, in some ways its mental model is more similar to an operating system than a web app.
+For example, say you make a Word doc, and then share it with someone else, who then makes revisions and sends it back to you via email.
+That’s analogous to a developer composing a Card, making it available for others to use, and accepting revisions. Someone could build an app _using_ Cardstack, but Cardstack itself is not an app.
```

Comment:
Should the language here be "authoring" a card?

Also, as I'm still wrapping my head around the card concept, should we be explicit about what we mean when we say accepting revisions? Are the revisions made to the data, to the implementation of the card, or both?

Reply:
Agreed on the wording change. I also do not have a clear picture of revisions, so I'll ping some of the other engineers to see how they would describe it.